packages/destination-actions/src/destinations/salesforce/sf-object-to-shape.ts

```diff
@@ -14,7 +14,16 @@ const LeadShape = (payload: GenericPayload): LeadBaseShapeType => {
   }
 }
 
-const objectToShape = new Map([['lead', LeadShape]])
+const CaseShape = (payload: GenericPayload): CaseBaseShapeType => {
+  return {
+    Description: payload.description
+  }
+}
+
+const objectToShape = new Map<string, (payload: GenericPayload) => any>([
```

Comment:
```suggestion
const objectToShape = new Map<string, (payload: GenericPayload) => LeadBaseShapeType | CaseBaseShapeType>([
```
Let's do this instead, it'll get clunky as we add actions but that's fine